src/App.vue

```diff
@@ -387,7 +387,7 @@ export default class App extends Mixins(
   }
   /** The Fee Summary filing text for firm businesses. */
   get filingLabelText (): string {
-    return this.isTypeFirm ? 'Dissolution' : ''
+    return this.isTypeFirm && this.isDissolutionFiling ? 'Dissolution' : ''
```

Comment:
I wonder if you should delete `isTypeFirm` from this?

Currently it looks like it might display "Dissolution" while registering a new firm, which would be incorrect.

Reply:
Please update the comment too. Thanks.

Reply:
I don't understand how this worked correctly before... It looks like this getter value was always used by SbcFeeSummary, so did the fee summary component always show "Dissolution" previously???

Reply:
Yes, it was a bug. This change will resolve it

Reply:
Would "null" be a better fallback value since we don't want the fee summary to use this prop (it's "not a value" rather than "empty string")?

Reply:
> I wonder if you should delete `isTypeFirm` from this?
> 
> Currently it looks like it might display "Dissolution" while registering a new firm, which would be incorrect.

I just remembered, the check for the FIRM type is required as if it is a dissolution for a non-firm company it should use the original filing text "Voluntary Dissolution"

Reply:
OK. Please add a comment saying this label text is specifically for firm dissolutions so we know next time ;)